releasenotes/notes/added-inplace-arg-8b7daf3abbd913cb.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    ``qiskit.result.utils.marginal_counts()`` now has a third argument
+    ``inplace`` which is ``False`` by default. Changing ``inplace=True`` 
+    operates on the original argument which can change the ``Result`` object 
+    passed to ``marginal_counts()``. This behaviour is limited
+    to ``Result`` object and does not affect if a dictionary is passed
+    as argument.
```

Comment:
```suggestion
    The function :func:`qiskit.result.utils.marginal_counts` now, by default, does not modify the 
    :class:`qiskit.result.result.Result` instance parameter.  A new parameter `inplace` in
     :func:`qiskit.result.utils.marginal_counts`  allows to conserve the previous behavior when set as
     `inplace=True`. This behaviour is limited
    to :class:`qiskit.result.result.Result` parameters and does not affect if a dictionary is passed
    as argument.
```

Reply:
Done! I guess I had to depend on you guys a lot. Thank You @taalexander and @1ucian0 for showing me the ropes! Hope to be able to adhere to the formats from now on. Thanks Again!